lib/discordrb.rb

```diff
@@ -32,15 +32,19 @@ module Discordrb
     server_message_typing: 1 << 11,
     direct_messages: 1 << 12,
     direct_message_reactions: 1 << 13,
-    direct_message_typing: 1 << 14
+    direct_message_typing: 1 << 14,
+    message_content: 1 << 15,
```

Comment:
I think maybe `server_message_content` may be a better name since the intent only effects message content in servers. DMs along with a [few other circumstances](https://discord.com/developers/docs/topics/gateway#message-content-intent) are unaffected.